src/index.ts

```diff
@@ -175,6 +177,14 @@ app.use(async (req, res, next) => {
         : noRights
     } as AuthData
   } catch (e) {
+    if (!axios.isAxiosError(e))
+      return () => {
+        console.error(
+          'user-service getMe failed with error is not AxiosError',
+          e
+        )
+        res.status(500).render('error-user-service')
+      }
```

Comment:
I think this is just a no-op since it returns a function but handler return values aren't utilized in express, right?